rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java

```diff
@@ -332,7 +341,9 @@ public void statsUpdated(Map<String, StatHistory> values) {
 
         StatHistory freeNodes = values.get("FreeNodesCount");
         StatHistory busyNodes = values.get("BusyNodesCount");
+        StatHistory deployingNodes = values.get("DeployingNodesCount");
```

Comment:
Maybe match the string values with the variable names freeNodes --> freeNodesCount ?, etc...

Reply:
I am not getting the idea. What do u want me to rename?